preupg/cli.py

```diff
@@ -54,126 +54,166 @@ def __init__(self, args=None):
+            help="Provide path to all-xccdf.xml of the set of modules which is"
+                 " to be used for assesing the system. By default, if there is"
+                 " just one set in %s, Preupgrade Assistant uses that one."
+                 " Option --scan works similarly." % settings.source_dir +
+                 " " * 40 +
+                 "Example: -c /usr/share/preupgrade/RHEL5_7/all-xccdf.xml"
         )
         self.parser.add_option(
-            "--riskcheck",
+            "-R", "--riskcheck",
             action="store_true",
             default=False,
-            help="Checks preupgrade assessment for INPLACE RISKS." + "\n"*15 +
-                 "Return values:" + "\n"*45 +
-                 "0 ... SLIGHT risk was detected." + "\n" * 25 +
-                 "1 ... MEDIUM, HIGH risks were detected." + "\n" * 20 +
-                 "2 ... EXTREME risk was detected."
-        )
-        self.parser.add_option(
-            "--force",
+            help="Return the highest reported level of risk or result related"
+                 " to system upgrade. Run Preupgrade Assistant first -"
+                 " assessment of the system needs to be performed before using"
+                 " this option. When this option is used in concert with"
+                 " --verbose option, summary of the risks are printed to"
+                 " STDOUT. If the --verbose option is used once, just HIGH and"
+                 " EXTREME risks are printed. If it is used twice, all the"
+                 " risks are printed. " + " " * 30 +
+                 "Return values:" + " " * 40 +
+                 "0 ... SLIGHT/MEDIUM risk or needs_inspection/fixed/"
+                 "informational/not_applicable/not_selected/not_checked/pass"
+                 " result" + " " * 50 +
+                 "1 ... HIGH risk or needs_action result" + " " * 25 +
+                 "2 ... EXTREME risk or error/fail result"
+        )
+        self.parser.add_option(
+            "-f", "--force",
             action="store_true",
             default=False,
-            help="Suppress user interaction"
+            help="Suppress user interaction."
         )
         self.parser.add_option(
-            "--text",
+            "-t", "--text",
             action="store_true",
             default=False,
-            help="Convert HTML results to text form by elinks, lynx or w3m"
+            help="Generate plain text assessment report alongside XML and HTML"
+                 " reports. The text report is converted from HTML using"
+                 " elinks, lynx or w3m tool."
         )
         self.parser.add_option(
-            "-v",
-            "--verbose",
+            "-v", "--verbose",
             action="count",
             default=0,
-            help="Verbose mode"
+            help="Show more information during the assessment."
         )
         self.parser.add_option(
-            "--cleanup",
+            "-C", "--cleanup",
```

Comment:
We have already "-c" option. It could be easily overlooked that this is "-C" (just case-sensitive change). I think that it should be skipped. It is not rule that each option has to have long and short form. In addition, maybe the "--contents" option will be removed in future, so "-c" will be freed later.